atom/browser/atom_browser_context.cc

```diff
@@ -60,10 +60,8 @@ class NoCacheBackend : public net::HttpCache::BackendFactory {
 
 std::string RemoveWhitespace(const std::string& str) {
   std::string trimmed;
-  if (base::RemoveChars(str, " ", &trimmed))
-    return trimmed;
-  else
-    return str;
+  return base::RemoveChars(str, " ", &trimmed) ?
```

Comment:
Personally not a fan of these ternary expressions, the if structure is much more explicit and easier to read IMO

Reply:
Hey, first of all Thanks for the review! Usually I would agree with you, but for some of the small functions (like the one you just reviewed), I believe that the ternary operator would make really no difference, and instead introduce shortness, at little or no cost to clarity. However, that is just my opinion.

Reply:
If you noticed, I left some of the large functions with huge if else statements alone, since then the ternary operator would just look like a random jumble of code, which would obviously reduce clarity and readability @MarshallOfSound

Reply:
The `? :` expression is usually used the the condition has no side effect, in this case `base::RemoveChars` did modify one of the returned variables and it makes code harder to understand by using `? :`.